10up/ElasticPress
# Settings screen improvements

### Description of the Change
This PR will merge the subscription name and id into one option along with some alignment improvements



Closes #2739

### How to test the Change

- Go to ElasticPress settings screen Dashboard
- Notice that the Subscription ID and Username are merged into one option
- Observe the proper alignment of options



### Changelog Entry

> Changed - Settings screen options

### Credits

Props @MARQAS 

### Checklist:
